ui/src/components/buttons/ButtonWithTooltip/index.tsx

```diff
@@ -17,9 +17,7 @@ const ButtonWithTooltip: React.FC<ButtonWithTooltipProps> = ({
 }) => (
   <Tooltip>
     <TooltipTrigger asChild>
-      <div className="h-full">
-        <Button {...props}>{children}</Button>
-      </div>
+      <Button {...props}>{children}</Button>
```

Comment:
Please be sure this change doesn't break UI where ButtonWithTooltip is being used already 🙏🏻

Reply:
It's used at 3 places. I have verified that the UI hasn't changed after this change.